### ISSUE TITLE
CloudFormation (CFN) Demo ROLL_BACK Error

### ISSUE THREAD

**Issue by AXRoux** [acantril/aws-sa-associate-saac03#3]
## ERROR:
```Parameter validation failed: parameter value for parameter name KeyName does not exist. Rollback requested by user.```

occurs when uploading the ``ec2instance.yaml`` file 

## SOLUTION

Remove lines 5, 6 and 7 from the .yaml file

```KeyName:```
    ```Type: AWS::EC2::KeyPair::KeyName```
    ```Description: "Name of an existing SSH Keypair to access the instance"```

**Comment by acantril** [comment]
Hi
You should have created a keyname A4L earlier in the course.
this is what this template should be using.

**Comment by acantril** [comment]
And when you say 'uploading'? you should be using 1-click deployments (attached to the lessons) not "uploading" a template.

**Comment by AXRoux** [comment]
Correct, I was confused because in the video there isn't a Key parameter so I just browsed over that part. Thanks for clearing up the confusion.

**Comment by acantril** [comment]
Because you are using the upload version - the demo doesn't use that.
You generally should be using the 1-click deployment links under the video in the lessons, not the versions (if any) on the github repo.
